tests/rados/test_osd_async_recovery.py

```diff
@@ -0,0 +1,105 @@
+import random
+import time
+
+from ceph.ceph_admin import CephAdmin
+from ceph.rados.core_workflows import RadosOrchestrator
+from tests.rados.monitor_configurations import MonConfigMethods
+from utility.log import Log
+from utility.utils import method_should_succeed
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    Performs the test osd sync tests.
```

Comment:
please check if the module is tests sync or async?

Reply:
Doc text is modified.